ROADMAP.md

```diff
@@ -15,9 +15,9 @@ For up to date release notes, refer to the project [Change Log](https://github.c
 
 ## Upcoming Releases
 
-### 3.6
+### 3.7
 
-- Release 3.6 will focus on adding support for the `@defer` directive as well as adding `schemaBased` test builders. See Github [3.6 Milestone](https://github.com/apollographql/apollo-kotlin/milestone/23) for more details.
+- Release 3.7 will focus on providing better Java support in Apollo Kotlin.  See Github [3.7 Milestone](https://github.com/apollographql/apollo-kotlin/milestone/25) for more details.
```

Comment:
I'd remove the specific `3.7` version here. There's a high chance we'll need to unbundle the Java runtime from the Kotlin one and therefore version numbers will not align

Reply:
So how would we communicate this release? Do you think it's going to a new version starting at 1.0 for Java?

Reply:
great. I left it as release `TBD` and we can decide the order of the releases as we break this work down further.